cmd/license.go

```diff
@@ -43,61 +42,54 @@ func extractLicenseIDs() error {
 		return err
 	}
 
-	// create two slices of license IDs, one for deprecated and one for not deprecated
+	// create two slices of license IDs, one for deprecated and one for active
+	var activeLicenseIDs []string
```

Comment:
This was `nonDeprecatedLicenseIDs.  Updated the var name to `activeLicenseIDs` as it is a better description of what the var holds.